src/components/Auth/Alert/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import bellIcon from 'images/icons/bell_icon.png';
+import errorIcon from 'images/icons/error_icon.png';
+
+const Info = ({ title, description }) => {
+  return (
+    <div className="row auth-alert">
+      <div className="col-2">
+        <div className="auth-alert__icon">
```

Comment:
That was my initial plan but then I thought that the error format will be different and might need to format that, so I made it like this so that the component doesn't get much complex. But I can make it into one component. What do you think? 🤔

Reply:
Yeah I think we can make it one and inherit from that 🤔

Reply:
@Lahphim  Made the changed 🚀